Vagrantfile

```diff
@@ -0,0 +1,49 @@
+Vagrant.configure("2") do |config|
+config.vm.box = "hashicorp/bionic64"
+    config.vm.network :forwarded_port, guest: 5000, host: 5000
+
+    config.vm.provision "shell", privileged: false, inline: <<-SHELL
+        sudo apt-get update
+
+        # Install pyenv prerequisites
+        sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+        libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+        xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+        
+        # Install pyenv
+        git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+
+        echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bash_profile
+        echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
```

Comment:
You shouldn't actually be updating both.

When you start a bash shell, it will look for .bash_profile and run it if it exists. If it doesn't exist, it will look for .profile.

A .bash_profile file did not exist until you created it in the provisioning script. If you just updated .profile it would have all worked as desired. 

And note that .profile was set up to detect whether you are running a bash shell and if so, it runs `.bashrc` as well. Your bash_profile is not set up with that, so you have now accidentally disabled some functionality.